### PR TITLE
data:dahlansatyam.json

### PR DESCRIPTION
**Pull request by dahlansatyam**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4693"></a>

### CODE REVIEW

**Review by himanshu1221 — APPROVED** [review]
LGTM :+1:

**Review by bharat5604 — DISMISSED** [review]
LGTM

**Review by priyankarpal — CHANGES_REQUESTED** [review]
hey @dahlansatyam you can add your all social links here.

**Comment by dahlansatyam** [comment]
@priyankarpal I'll add links sometime later.

**Review by priyankarpal — APPROVED** [review]

**Review by krshkun — COMMENTED** [review]
Looks good but maybe separate the change inthe code from the profile

File: components/user/UserTabs.js [hunk 1]
```diff
@@ -93,7 +93,7 @@ export default function UserTabs({ tabs, setTabs, setUserData, userData }) {
           name="tabs"
           onChange={changeTab}
           className="block w-full rounded-md border-gray-300 focus:border-indigo-500 focus:ring-indigo-500"
-          defaultValue={tabs.find((tab) => tab.current).name}
+          defaultValue={tabs.find((tab) => tab.current.name)}
```
Comment (amandamartin-dev) [inline components/user/UserTabs.js:96]: Confused about the change in this component as the PR is only for updating your profile.  Should this be here?

**Review by eddiejaoude — DISMISSED** [review]
Please keep this PR focused on your profile only

File: components/user/UserTabs.js (hunk 1 above)
Comment (dahlansatyam) [reply components/user/UserTabs.js:96]: I'll separate this in the new PR.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please revert the changes in the `package-lock.json` file.
Let us know if you need any help with that. 🙂

**Comment by dahlansatyam** [comment]
please have a look now @Panquesito7

**Review by Panquesito7 — COMMENTED** [review]
Thanks! 🎉

**Review by Panquesito7 — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**